test/specs/test-error-list.el

```diff
@@ -137,9 +137,8 @@
       (it "has the error ID in the 4th cell"
         (expect (aref cells 3) :to-equal
                 (list "W1"
-                      'type 'flycheck-error-list
-                      'face 'flycheck-error-list-id
-                      'help-echo "W1")))
```

Comment:
Previously it was the error id, now it's "mouse 2, RET: explain error".  Since this text was not tested for the other cells where it appears (line, column, level), I removed it here as well.